third-party/chromium-synchronization/inspector-issueAdded-types-test.js

```diff
@@ -13,8 +13,6 @@ const inspectorIssuesGathererPath = LH_ROOT +
   '/lighthouse-core/gather/gatherers/inspector-issues.js';
 const inspectorIssuesGathererSource = fs.readFileSync(inspectorIssuesGathererPath, 'utf-8');
 
-/* eslint-env jest */
```

Comment:
We're removing this comment but I can't find an associated eslintrc file added for this directory? Is there already one higher up?

Reply:
eslintignore ignores the entire third-party folder